library_generation/README.md

```diff
@@ -0,0 +1,91 @@
+# Generate GAPIC Client Library without post-processing
+
+The script, `generate_library.sh`, allows you to generate a gapic client library from proto files.
+
+## Environment
+
+Use Linux environment and install java runtime environment (8 or above).
+
+## Prerequisite
+In order to generate a GAPIC library, you need to pull `google/` from [googleapis](https://github.com/googleapis/googleapis)
+and put it into the directory containing `generate_library.sh` since protos in `google/` 
+are likely referenced by protos from which the library are generated.
+
+In most cases, the `proto_path` is a subdirectory within `google/`.
+
+## Parameters to run `generate_library.sh`
+
+You need to run the script with the following parameters.
+
+### proto_path
+A directory in the current working directory (refers as `$cwd`) and copy proto files into it. 
+The absolute path of `proto_path` is `$cwd/$proto_path`. 
+
+Use `-p` or `--proto_path` to specify the value.
+
+   Note that usually additional proto files are needed to generate a gapic library, 
+   you should also copy `google/` folder from [googleapis](https://github.com/googleapis/googleapis) 
+   into `$cwd`.
+
+### destination_path 
+A directory in `$cwd`. This is the path in which the generated library will reside. 
```

Comment:
The usage is unclear to me. What value should a user specify here as the destination? (I see google-cloud-confidentialcomputing-v1-java example below but this directory does not exist in the java-confidentialcomputing module.)


Can you add how user will use the generated library? "We will use this destination to do ..."

Reply:
The generated library is under `$destination_path`, the directory structure is
```
$destination_path
  |_gapic-*
  |    |_src
  |       |_main
  |          |_java
  |          |_resources
  |       |_test
  |_grpc-*
  |    |_src
  |       |_main
  |          |_java
  |    
  |_proto-*
       |_src
          |_main
             |_java
             |_proto
```

This is how pure generated library looked like in `googleapis-gen`.

This PR will generate the library without post processing, so there's no pom.xml, bom, etc. I think it can't be used by users.

Reply:
The directory structure explanation is great and worth adding here. So users will copy the generated files to the corresponding directories of a library repository.

Reply:
done

Reply:
Thank you